vit-servicing-station-lib/src/db/models/snapshot.rs

```diff
@@ -65,24 +65,24 @@ pub struct Contributor {
     pub snapshot_tag: String,
 }
 
-impl Insertable<contributors::table> for Contributor {
+impl Insertable<contributions::table> for Contribution {
```

Comment:
This looks derivable, can't see anything funky in the impl

Reply:
Sorry, was my fault. Yes you are right, updated